core/templates/dev/head/pages/learner_dashboard/LearnerDashboard.js

```diff
@@ -301,22 +301,20 @@ oppia.controller('LearnerDashboard', [
       constants.ACTIVITY_TYPE_EXPLORATION);
 
     $scope.onClickThread = function(
-        threadStatus, explorationId, threadId, explorationTitle) {
+        threadStatus, threadId, explorationTitle) {
       $scope.loadingFeedbacks = true;
       var threadDataUrl = UrlInterpolationService.interpolateUrl(
-        '/learnerdashboardthreadhandler/<explorationId>/<threadId>', {
-          explorationId: explorationId,
+        '/learnerdashboardthreadhandler/<threadId>', {
           threadId: threadId
         });
       $scope.explorationTitle = explorationTitle;
       $scope.feedbackThreadActive = true;
       $scope.threadStatus = threadStatus;
-      $scope.explorationId = explorationId;
+      $scope.explorationId = threadId.split('.')[0];
```

Comment:
Doesn't generalize. Should we be passing the explorationId as a separate arg instead? The less dependence we have on computing stuff based on a thread ID (which should be treated by clients as an arbitrary string), the better.

Reply:
Passed explorationId instead.